packages/hash/shared/src/json-utils.ts

```diff
@@ -17,6 +17,7 @@ export const parseJson = <T extends JSONObject | JSONArray>(
 
 export type JsonSchema = Schema & {
   $defs?: Record<string, JsonSchema>;
+  default?: JSONValue;
```

Comment:
This might be a stupid question, but why do we define `default` not in the same place as `variants`?

Reply:
It's a reasonable question. `default` is a JSON schema specified keyword (albeit not covered by this library's types) so it seems natural to put it in the schema (a bit more in [this comment](https://github.com/hashintel/hash/pull/481#discussion_r848613082)).

We could potentially put `default` in `block-metadata.json` also, or put `variants` in the schema file.

I'm not sure about putting `variants` in the schema file, since it seems closer to metadata about the block (what variants should be displayed as selectable and so on) rather than being schema-specific – although the distinction is a bit fuzzy, as `variants` should be compliant with the schema.